shared/src/commonMain/kotlin/org/hyperskill/app/project_selection/presentation/ProjectSelectionListStateExtensions.kt

```diff
@@ -0,0 +1,63 @@
+package org.hyperskill.app.project_selection.presentation
+
+import org.hyperskill.app.progresses.domain.model.averageRating
+import org.hyperskill.app.project_selection.presentation.ProjectSelectionListFeature.ContentState
+import org.hyperskill.app.projects.domain.model.ProjectLevel
+import org.hyperskill.app.projects.domain.model.ProjectWithProgress
+import org.hyperskill.app.track.domain.model.getProjectsIds
+
+internal val ContentState.Content.selectedProject: ProjectWithProgress?
```

Comment:
`ProjectSelectionListFeature.State` is already marked as `internal`, is it necessary to mark it here with `internal` modifier?

Reply:
It's impossible to expose an internal type via the public member.
In this case extension receiver is internal and it's not allowed to create a public extension on it. It could only be internal.

Reply:
OK, thanks